pkg/setting/setting.go

```diff
@@ -755,6 +756,7 @@ func (cfg *Cfg) Load(args *CommandLineArgs) error {
 	}
 
 	// admin
+	DisableAdminUser = security.Key("disable_admin_user").MustBool(false)
```

Comment:
Here, instead of using public variable `DisableAdminUser`, assign it to `cfg.DisableAdminUser` and move the variable `DisableAdminUser` into `Cfg` struct.